app/views/coronavirus_pages/index.html.erb

```diff
@@ -2,15 +2,15 @@
 
 <h2 class="govuk-heading-s govuk-!-margin-bottom-2">Coronavirus landing page</h2>
 <ul class="govuk-list">
-  <li><%= link_to "Edit landing page accordions and announcements", coronavirus_page_path(slug: @topic_page.slug), class:"govuk-link" %></li>
-  <li><%= link_to "Edit live stream URL", live_stream_index_path, class:"govuk-link" %></li>
-  <li><%= link_to "Edit something else on the landing page", prepare_coronavirus_page_path(slug: @topic_page[:slug]), class:"govuk-link" %></li>
+  <li><%= link_to t("coronavirus_pages.index.landing_page_edit.accordians"), coronavirus_page_path(slug: @topic_page.slug), class:"govuk-link" %></li>
+  <li><%= link_to t("coronavirus_pages.index.landing_page_edit.live_stream_url"), live_stream_index_path, class:"govuk-link" %></li>
+  <li><%= link_to t("coronavirus_pages.index.landing_page_edit.something"), prepare_coronavirus_page_path(slug: @topic_page[:slug]), class:"govuk-link" %></li>
```

Comment:
For this and the following one I'd have thought `something_else` would be a more appropriate key as `something` sounds like the opposite.

Reply:
Thanks! Have updated.